### PR TITLE
Accesses "cost history" in Cloud Health API for current month

### PR DESCRIPTION
**Pull request by khhirani**
I created CloudAPI.py, which accesses the Cloud Health API with the user's inputted API key and prints the "cost history" information for the current month. 

The next step is to create a program which allows the user to access different parts of the API and use filters, time ranges, etc..

### CODE REVIEW

File: krux_cloud_health/CloudAPI.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+        group.add_argument(
+            '--api-key',
+            help="API key to retrieve data",
+        )
+
+    def get_report(self, report, api_key):
+        uri = urlparse.urljoin(API_ENDPOINT, report)
+        # uri += "?" + time_filter
+        uri += "?api_key=%s" % api_key
```
Comment (hanpeter) [inline krux_cloud_health/CloudAPI.py]: Nit: `requests` has a way to handle query strings: http://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls

This is not a show stopper for this PR, but we should fix this before production.

**Comment by hanpeter** [comment]
Nit: Please add empty lines at the end of the files.

Otherwise, LGTM.

**Comment by dpricha89** [comment]
👍

File: krux_cloud_health/CloudAPI.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    def get_report(self, report, api_key):
+        uri_args = {'api_key': api_key}
+        uri = urlparse.urljoin(API_ENDPOINT, report)
+        r = requests.get(uri, params=uri_args)
+        return r.json()
+
+    def run(self):
+        call = self.get_report("cost/history", self.api_key)
+
+        curr_month = "2016-06"
```
Comment (antonwinter) [inline krux_cloud_health/CloudAPI.py:67]: I doubt this is meant to be hard coded.

**Pull request merged**